docs/table/access_table.rst

```diff
@@ -374,6 +374,49 @@ For columns the syntax and behavior of
   2973.0
   Length = 100 rows
 
+  Column alignment
+''''''''''''''''
```

Comment:
The extra spaces before "Column alignment" are the culprit.

The log gives the hint, here: 
```
/home/travis/build/astropy/astropy/docs/table/access_table.rst:378: WARNING: Literal block ends without a blank line; unexpected unindent.
```